GliaWidgets/Sources/Coordinators/Chat/ChatCoordinator.Environment.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
```

Comment:
You mentioned previously about the issue with SPM

> Swift Package Manager can’t resolve path to files that located inside of folder with a name that has dots, for example: Glia.Environment. To avoid this issue, please omit using dots in folder names

Reply:
> dots in **folder** names
I mentioned do it folder names, but in files it is OK to use.

Reply:
Ohh, misread) Sorry)